### PR TITLE
create episode + episode controller + season + season controller

### PR DESCRIPTION
**Pull request by clounarae**
#4

### CODE REVIEW

File: src/controllers/EpisodeController.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from src.models.Episode import *
+
+
+class EpisodeController:
+
```
Comment (nicolay1) [inline src/controllers/EpisodeController.py]: You may add a comment to describe the class

File: src/models/Episode.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+from typing import List
+
+
+class Episode:
+
```
Comment (nicolay1) [inline src/models/Episode.py]: You may add a comment to describe the class :) you may also check the types :)

File: src/models/Episode.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            raise TypeError("Episode number must be an integer")
+        else:
+            self.__num_ep = value
+
+    @property
+    def summary(self):
+        return self.__summary
+
+    @summary.setter
+    def summary(self, value):
```
Comment (nicolay1) [inline src/models/Episode.py]: what is the type of value ?

File: src/models/Episode.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            raise TypeError("Season Id must be an integer")
+        else:
+            self.__id_season = value
+
+    @property
+    def num_ep(self):
+        return self.__num_ep
+
+    @num_ep.setter
+    def num_ep(self, value):
```
Comment (nicolay1) [inline src/models/Episode.py]: what is the type of value ?

File: src/models/Episode.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            raise TypeError("Name should be a string")
+        else:
+            self.__name = value
+
+    @property
+    def id_season(self):
+        return self.__id_season
+
+    @id_season.setter
+    def id_season(self, value):
```
Comment (nicolay1) [inline src/models/Episode.py]: what is the type of value ?

File: src/models/Season.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
+from typing import List
+from src.models.Episode import *
+
+
+class Season:
+
```
Comment (nicolay1) [inline src/models/Season.py]: please add a class comment :)

File: src/models/Season.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+            raise TypeError("Show id must be an integer")
+        else:
+            self.__id_show = value
+
+    @property
+    def num_season(self):
+        return self.__num_season
+
+    @num_season.setter
+    def num_season(self, value):
```
Comment (nicolay1) [inline src/models/Season.py]: please add a type :)

File: src/models/Season.py [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+            raise TypeError("Season number must be an integer")
+        else:
+            self.__num_season = value
+
+    @property
+    def list_episodes(self):
+        return self.__list_episodes
+
+    @list_episodes.setter
+    def list_episodes(self, value):
```
Comment (nicolay1) [inline src/models/Season.py]: please add a type :)

File: src/models/Episode.py [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+        self.__summary = summary
+        self.__list_actor = list_actor
+        self.__list_author = list_author
+
+    @property
+    def name(self):
+        return self.__name
+
+    @name.setter
+    def name(self, value):
```
Comment (nicolay1) [inline src/models/Episode.py]: please set a type for value :)

File: src/controllers/SeasonController.py [hunk 10]
```diff
@@ -0,0 +1,8 @@
+from src.models.Season import *
+
+
+class SeasonController:
+
```
Comment (nicolay1) [inline src/controllers/SeasonController.py]: Please add a comment to describe the class

File: src/models/Episode.py [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+            raise TypeError("Summary should be a string")
+        else:
+            self.__summary = value
+
+    @property
+    def list_actor(self):
+        return self.__list_actor
+
+    @list_actor.setter
+    def list_actor(self, value):
```
Comment (A-Chapuis) [inline src/models/Episode.py]: Precise the type here too ?

File: src/models/Episode.py [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+            raise TypeError("Actor's list should be a list")
+        else:
+            self.__list_actor = value
+
+    @property
+    def list_author(self):
+        return self.__list_author
+
+    @list_author.setter
+    def list_author(self, value):
```
Comment (A-Chapuis) [inline src/models/Episode.py]: Type here too ?

File: src/models/Season.py [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+        self.__id_show = id_show
+        self.__num_season = num_season
+        self.__list_episodes = list_episodes
+
+    @property
+    def id_show(self):
+        return self.__id_show
+
+    @id_show.setter
+    def id_show(self, value):
```
Comment (A-Chapuis) [inline src/models/Season.py]: Type here too ?

**Pull request merged**